docs/userGuide/userQuickStart.md

```diff
@@ -132,3 +133,5 @@ By default, it will try to push everything in the generated site (default dir: `
 More details of deployment setting could be found in [here](ghpagesDeployment.html).
 
 </div>
+
+<include src="../_markbind/footers/userGuideSections.md" />

```

Comment:
Shouldn't have this line since the footer is already specified in the frontmatter. Anyway we are getting rid of `userGuideSections.md` right?

Reply:
Leftovers from rebasing master